conda/core/package_cache.py

```diff
@@ -226,11 +244,12 @@ def get_entry_to_link(cls, dist):
         raise CondaError("No package '%s' found in cache directories." % dist)
 
     @classmethod
-    def tarball_file_in_cache(cls, tarball_path, md5sum=None):
+    def tarball_file_in_cache(cls, tarball_path, md5sum=None, exclude_caches=[]):
```

Comment:
Yes, exactly.  I know python programmers who, if running across that, would instantly judge the whole code base as crap.  I'm sure there are a lot of blog post rants about it too.  Maybe google something like "python list as default argument".